### PR TITLE
Folderpluginupdates

### PR DESCRIPTION
**Pull request by theClueless**
Program plugin changes:
- support for share folders (start with "\\")
- support for basic context menus like everything plugin

### CODE REVIEW

File: Plugins/Wox.Plugin.Folder/Main.cs [hunk 1]
```diff
@@ -59,6 +65,26 @@ public List<Result> Query(Query query)
...
         }
 
+        private static bool IsDriveOrSharedFolder(string search)
+        {
+            if (search.StartsWith(@"\\"))
+            { // share folder
+                return true;
+            }
+
+            if (_driverNames != null && _driverNames.Any(search.StartsWith))
```
Comment (jjw24) [inline Plugins/Wox.Plugin.Folder/Main.cs:75]: search.StartsWith - is this correct with no value provided?

**Comment by jjw24** [comment]
These are great enhancements, by the rate you are going we might need to rename this plugin to Windows Explorer :P

File: Plugins/Wox.Plugin.Folder/Main.cs (hunk 1 above)
Comment (theClueless) [reply Plugins/Wox.Plugin.Folder/Main.cs:75]: This is the same code as before, not sure what you mean by no value.
like -> search = " " ?
or -> search = "c" ?
or when you change the action keyword? and then get an empty value? but even so how is that an issue?

Comment (jjw24) [reply Plugins/Wox.Plugin.Folder/Main.cs:75]: all good never mind. Didnt know linq's expression _driverNames.Any(search.StartsWith) is same as _driverNames.Any(x => search.StartsWith(x)). Thats why i was wondering why StartsWith has no parameter passed.

**Review by jjw24 — APPROVED** [review]

**Pull request merged**